content/post/AG2021.en.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "2021 general assembly review"
+date: 2021-09-23
+draft: false
+tags: [general-assembly]
+---
+
+Like every year, we hold our general assembly. On the 4th of September, we were gathered in the Nantes area, with a few remote members, to look back upon the past year and decide on the association stances for the year to come.
+
+### A year dampened by the Covid pandemic
+
+Same as everyone, we saw our outside activity slow down; we spoke at less conferences, but still had as many interviews with journalists.
+
+Less conferences and workshops doesn't mean **no** conferences or workshops. We had a few meaningful encounters:
+
+- at Lycée Blaise Pascal in Rouen, with the class of seconde 8 in SNT (Digital sciences and technology) lesson, and their teacher Yves Meret. This speech was briefly featured in the France 2 show Envoyé Spécial on 29/04/2021;
+- an interview with U+039b, one of the founders of the association, in the other France 2 show Cash Investigation;
+- participating in the France Inter podcast with Xavier de La Porte, "Le code a changé" on 07/12/2020, broadcasted on France Inter in summer 2021;
+- a lecture in the Foyer des Jeunes Travailleurs in Batignoles...
```

Comment:
```suggestion
- a lecture in the Foyer des Jeunes Travailleurs in Batignoles…
```

Reply:
Actually why is an ellipsis needed here? Is it a list that's not exhaustive?

Reply:
I guess so, as we met other people than what we list here!